### ISSUE TITLE
DP - Object detection 

### ISSUE THREAD

**Issue by JpMCarrilho** [tensorflow/privacy#94]
Hi, is it possible to use DPSGD Optimizer directly in an Object Detection Architecture? And if so, would that make my model diffrentially private (coonsidering the right combination of hyper-parameters)?

**Comment by galenmandrew** [comment]
I don't have any familiarity with Object Detection Architecture, but if it has a loss function that you can minimize with SGD, then training it with a DPSGD Optimizer would indeed yield a differentially private model.

**Comment by npapernot** [comment]
feel free to reopen if this did not answer your question @JpMCarrilho
